model-optimizer/extensions/ops/gather.py

```diff
@@ -43,26 +61,31 @@ def infer(node: Node):
         assert data_shape is not None
         indices_shape = node.in_port(1).data.get_shape()
         assert indices_shape is not None
-        axis = node.in_port(2).data.get_value()
-        assert axis is not None
-        axis = get_canonical_axis_index(data_shape, axis)
+        assert node.in_port(2).data.get_value() is not None
```

Comment:
Add an error message with the node name.
The same comments to other places where `assert` present

Reply:
I lowered exposure on this comment to Nit and I leaved this assert untouched.

Indeed we do not have unified approach for checking if inputs have valid shapes/values. In one place Error is raised in other asserts https://github.com/openvinotoolkit/openvino/blob/master/model-optimizer/extensions/ops/gathernd.py#L30

If I change this assert then we need to change in a lot of other places for unification.